book/02-git-basics/sections/aliases.asc

```diff
@@ -1,5 +1,5 @@
 [[_git_aliases]]
-=== Git Takma Adları
+=== Git Aliases
```

Comment:
I think you didn't understand. This part is a title and it must be translated.

OTOH, you mustn't translate the lines `[source,console]`

Reply:
sorry :) I'm fixing now